data/modules/Combat/Combat.lua

```diff
@@ -140,29 +140,35 @@ local onChat = function (form, ref, option)
 		end
 
 	elseif option == 5 then
-		form:RemoveAdvertOnClose()
-		ads[ref] = nil
-		local mission = {
-			type        = "Combat",
-			client      = ad.client,
-			faction     = Game.system.faction.id,
-			org         = ad.org,
-			location    = ad.location,
-			rendezvous  = ad.rendezvous,
-			mercenaries = {},
-			introtext   = ad.introtext,
-			flavour     = ad.flavour,
-			in_progress = false,
-			complete    = false,
-			risk        = ad.risk,
-			dedication  = ad.dedication,
-			reward      = ad.reward,
-			bonus       = -(math.ceil(ad.dedication * 3) - 1),
-			due         = ad.due,
-		}
-		table.insert(missions,Mission.New(mission))
-		form:SetMessage(l["ACCEPTED_" .. Engine.rand:Integer(1, getNumberOfFlavours("ACCEPTED"))])
-		return
+		if Game.player:GetEquip('radar', 1) then
```

Comment:
Rather than indenting the entire block, I'd prefer you simply invert the if statement to check if the player *doesn't* have a radar and early out, leaving the block at the existing indentation.

Reply:
I hope I got it right